### ISSUE TITLE
Remove fixuid from the docker image

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fyne-cross-images#44]
The fyne-cross tool has removed the need for using fixuid. We should remove it sometime going forward. Might require new versions as this would be a breaking change for someone using the old tools.

**Comment by Bluebugs** [comment]
I think this is a very low priority as this will be a breaking change which require bumping up fyne-cross major release and the image too. Making also sure that old version won't pick the new version. I don't see a good reason to go through all this burden.

**Comment by Jacalz** [comment]
Yeah, I understand. I simply opened the issue to keep track of it so it doesn't get lost and forgotten.
